frontend/app/src/services/history/history-api.ts

```diff
@@ -164,6 +165,8 @@ export class HistoryApi {
           ...data,
           orderByAttribute: getUpdatedKey(payload.orderByAttribute ?? '', false)
         }),
+        paramsSerializer: params =>
+          Qs.stringify(params, { arrayFormat: 'repeat' }),
```

Comment:
@lukicenturi this dependency is only used here to format the array in the query string in the repeated format right?

@LefterisJP we are not consistent here, I remember in some places we changed the endpoint to POST to just pass this as a body, in other places I remember that we used to send comma separated values. Can we also change the endpoint here to POST?

Reply:
Updated